### ISSUE TITLE
All jianshu.com links have "404 not found" error.

### ISSUE THREAD

**Issue by hlvlad** [yjjnls/awesome-blockchain#18]
Sadly, all links to jianshu.com site from "Basic Introduction" section are broken.

**Comment by yjjnls** [comment]
Yes, jianshu site has disabled all blogs with link to github. Later I will republish these works in other blog site

**Comment by progmatic-99** [comment]
@yjjnls Can you republish these blogs, I want to understand those concepts.
Thanks for your help.

**Comment by roseJoke** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by sfzygit** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我已收到您的邮件！

**Comment by parth35** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

Lawliet-Chan, @RustMan88, @CNFINITE, @Taylo0or, @danerlt, @iocn-io, @gexin1023, @swlfigo, @weibin159, @jason7580, @foreseaz, @wangkuanzzu, @onemedicine, @06wagon, @colindog, @CodingCattwo, @zhaojingyu, @trevorfu, @WAAutoMaton, @chen-si

**Comment by roseJoke** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by sfzygit** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我已收到您的邮件！
